modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+
+* Before this update, when using {pac}, if you set the `concurrency_limit` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` or `pipelines-as-code` namespace, which provides default settings for all `Repository` CRs, the {pac} watcher crashed. With this update, the {pac} watcher operates correctly with this setting.
+
+* Before this update, all tasks in {pipelines-shortname} included an extra step compared to the cluster tasks of the same name that were available in previous versions of {pipelines-shortname}. This extra step increased the load on the cluster. With this update, the tasks no longer include the extra step as it is integrated into the first step.
+
+* Before this update, when you used one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the parameter was not interpreted correctly and the task failed. With this update, the task interprets the `CONTEX`T parameter correctly and completes successfully.
+
+* Before this update, in {tekton-chains} the in-toto provenance metadata, `URI` and `Digest`, were incomplete. They contained only the information of remote `Pipeline` and `Task` resources, but were missing the information of the remote `StepAction` resource. With this update, the provenance of the remote `StepAction` resource is recorded in the task run status and inserted into the in-toto provenance, which results in complete in-toto provenance metadata.
+
+* Before this update, you could modify some of the parameters in the `spec` field of the `PipelineRun` and `TaskRun` resources that should not be modifiable after the resources were created. With this update, you can only modify the allowed fields after the pipeline run and task run are created, such as `status` and `statusMessage` fields.
+
+* Before this update, if a step action parameter was an `array` type but a `string` value was passed in a task, there was no error indicating inconsistent parameter types and the default parameter value was used instead. With this update, an error is added to indicate the inconsistent values: `invalid parameter substitution: %s. Please check the types of the default value and the passed value`.
+
+* Before this update, task runs and pipeline runs were deleted by an external pruner when logs were streamed through the watcher. With this update, a finalizer is added to {tekton-results} for `TaskRun` and `PipelineRun` objects to ensure that the runs are stored and not deleted. The runs are stored either as records or until the deadline has passed, which is calculated as the completion time plus the `store_deadline` time. The finalizer does not prevent deletion if legacy log streaming from the watcher or pruner is enabled. 
```

Comment:
None of these bugs link to Jiras. The core OCP RN doc would include links so customers can explore further. From a docs perspective, the links help find duplicates and so forth.

Reply:
I had the same feedback.

Reply:
This is just how Pipelines RNs work for several years now.